app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsFormModal/LocalUnitsForm/index.tsx

```diff
@@ -592,6 +660,15 @@ function LocalUnitsForm(props: Props) {
                                 && (environment !== 'production')
                                 && (
                                     <div className={styles.actions}>
+                                        {localUnitDetailsResponse?.is_locked && (
```

Comment:
Let's drop optional chaining.

Also, shouldn't the logic be inverted? Do we show "revert" button if the localUnit is locked?

Reply:
yes, we do show revert button if localunit is locked